crates/meilisearch/tests/search/filters.rs

```diff
@@ -10,62 +10,63 @@ use crate::{
 
 #[actix_rt::test]
 async fn search_with_filter_string_notation() {
-    let server = Server::new().await;
-    let index = server.index("test");
+    let server = Server::new_shared();
+    let index = server.unique_index();
 
     let (_, code) = index.update_settings(json!({"filterableAttributes": ["title"]})).await;
-    meili_snap::snapshot!(code, @"202 Accepted");
+    snapshot!(code, @"202 Accepted");
 
     let documents = DOCUMENTS.clone();
     let (task, code) = index.add_documents(documents, None).await;
-    meili_snap::snapshot!(code, @"202 Accepted");
-    let res = index.wait_task(task.uid()).await;
-    meili_snap::snapshot!(res["status"], @r###""succeeded""###);
+    snapshot!(code, @"202 Accepted");
+    let res = index.wait_task(task.uid()).await.succeeded();
+    snapshot!(res["status"], @r###""succeeded""###);
 
     index
         .search(
             json!({
                 "filter": "title = Gläss"
             }),
             |response, code| {
-                assert_eq!(code, 200, "{}", response);
+                assert_eq!(code, 200, "{response}");
                 assert_eq!(response["hits"].as_array().unwrap().len(), 1);
             },
         )
         .await;
 
-    let index = server.index("nested");
+    let nested_index = server.unique_index();
 
-    let (_, code) =
-        index.update_settings(json!({"filterableAttributes": ["cattos", "doggos.age"]})).await;
-    meili_snap::snapshot!(code, @"202 Accepted");
+    let (_, code) = nested_index
+        .update_settings(json!({"filterableAttributes": ["cattos", "doggos.age"]}))
+        .await;
+    snapshot!(code, @"202 Accepted");
 
     let documents = NESTED_DOCUMENTS.clone();
-    let (task, code) = index.add_documents(documents, None).await;
-    meili_snap::snapshot!(code, @"202 Accepted");
-    let res = index.wait_task(task.uid()).await;
-    meili_snap::snapshot!(res["status"], @r###""succeeded""###);
+    let (task, code) = nested_index.add_documents(documents, None).await;
+    snapshot!(code, @"202 Accepted");
+    let res = nested_index.wait_task(task.uid()).await.succeeded();
+    snapshot!(res["status"], @r###""succeeded""###);
```

Comment:
And here we can use the index with nested documents https://github.com/meilisearch/meilisearch/blob/ea6bb4df1d76e8ae3880b60a99b85b1102891a68/crates/meilisearch/tests/common/mod.rs#L344

Reply:
Here the difference is - the shared index uses these settings `{"filterableAttributes": ["father", "doggos"], "sortableAttributes": ["doggos"]}` and the index in the test: `{"filterableAttributes": ["cattos", "doggos.age"]}`